docs/index.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Joy of Coding 2023 - June 23rd</title>
+  <title>Joy of Coding 2024</title>
```

Comment:
Dit gaat later mogelijk problemen leveren. 
Alle pagina's van 2023 moeten zonder wijziging (behalve hrefs van parent/child pagina's) moeten anar jaar 2023 verplaatst worden. 
Vervolgens maken we een kopie daarvan naar root. En dan beginnen we vanaf nieuw.

Reply:
Goed punt, ik heb net de hele 2023 site gekopieerd vanaf master.